common/unsafe/src/test/java/org/apache/spark/unsafe/types/CalendarIntervalSuite.java

```diff
@@ -18,6 +18,7 @@
 package org.apache.spark.unsafe.types;
 
 import org.junit.Test;
+import scala.Int;
```

Comment:
Does this need to be imported?

Reply:
Otherwise compilation fails with:
```
[error] /Users/maxim/proj/interval-mul-div/common/unsafe/src/test/java/org/apache/spark/unsafe/types/CalendarIntervalSuite.java:318:  error: cannot find symbol
[error]       interval.multiply(Int.MaxValue());
[error]                         ^
```

Reply:
Weird, I don't think `scala.Int` has to be imported elsewhere; it's automatic. `Int.MaxValue`?

Reply:
oh, this is java. I will replace it by `Integer.MAX_VALUE`

Reply:
Doh I didn't even see that. Yes you are right